sources/web/datalab/polymer/components/datalab-app/datalab-app.ts

```diff
@@ -125,6 +125,29 @@ class DatalabAppElement extends Polymer.Element {
     }
   }
 
+  /**
+   * Shows a notification toast at the bottom of the page with the given message.
+   * @param message message to show in the notification toast
+   * @param sticky whether the toast should stick around until explicitly dismissed in code
+   */
+  showNotification(message: string, sticky?: boolean) {
```

Comment:
Given that you are only using this function to show connection methods, and you clear the toast whenever a connection succeeds, I think this should probably be called showConnectionNotification. Or maybe have a ConnectionToast component.